en/docs/develop/test/execute-tests.md

```
@@ -1,229 +1,237 @@
-// Runs before every test
-@test:BeforeEach
-function beforeEach() {
-    io:println("Setting up individual test");
-}
+## Run all tests
 
-@test:Config {groups: ["integration"]}
-function testOrderCreation() {
-    test:assertTrue(true, msg = "Order creation failed");
-}
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)